### DISCUSSION TITLE
Converting an HTML page from a website to a note in Obsidian

### DISCUSSION THREAD

**Discussion by somelinuxnerd** [BryanJenksCommunity/FAQ#82] [Q&A] [answered]
### Question

What is your question?

I would like some way to copy content from a webpage and paste into into an Obsidian note and maintain the formatting, primarily for tables of data. Is there a particular plugin for this that would be useful?

---

By filing a question to this repo, I promise that

-   [x] I have searched open/closed issues on this repo for my answer already

I understand that my issue may be closed if I don't fulfill my promises.

**Comment by tallguyjenks** [comment, marked as answer]
there's a feature in the obsidian settings that allows you to paste HTML content so that you can copy content directly from a page and paste it straight into obsidian :)

**Reply by tonymynd** [reply]
Thanks bro, how do we that? By a search in the plugins?

**Reply by tallguyjenks** [reply]
it should be in the standard apps settings

**Comment by ajitgoel** [comment]
Would it be this setting? I tried by turning it both off/on and then using cmd+shift+v to paste a table and it does not work. 

![Pasted_Image_3_17_23__11_21_AM](https://user-images.githubusercontent.com/6394444/225987395-135ddf97-ba33-46f0-94a3-4ae778ba7460.jpg)

**Reply by tallguyjenks** [reply]
yup!
